ejb3/src/main/resources/schema/jboss-ejb-client_1_3.xsd

```diff
@@ -0,0 +1,330 @@
+                        Configures the connection creation options for the node
+                    </xsd:documentation>
+                </xsd:annotation>
+            </xsd:element>
+        </xsd:choice>
+        <xsd:attribute name="name" type="xsd:string" use="required">
+            <xsd:annotation>
+                <xsd:documentation>
+                    The name of the cluster node
+                </xsd:documentation>
+            </xsd:annotation>
+        </xsd:attribute>
+        <xsd:attribute name="connect-timeout" type="xsd:long" use="optional">
+            <xsd:annotation>
+                <xsd:documentation>
+                    The timeout in milliseconds while creating a connection for the node
+                </xsd:documentation>
+            </xsd:annotation>
+        </xsd:attribute>
+        <xsd:attribute name="username" type="xsd:string" use="optional">
+            <xsd:annotation>
+                <xsd:documentation>
+                    The username that will be used for authentication during connection creation for the node
+                </xsd:documentation>
+            </xsd:annotation>
+        </xsd:attribute>
+        <xsd:attribute name="security-realm" type="xsd:string" use="optional">
+            <xsd:annotation>
+                <xsd:documentation>
+                    The security-realm that will be used for authentication during connection creation for the node
+                    cluster
+                </xsd:documentation>
+            </xsd:annotation>
+        </xsd:attribute>
+    </xsd:complexType>
+
+    <xsd:complexType name="channel-creation-optionsType">
+        <xsd:choice minOccurs="0" maxOccurs="unbounded">
+            <xsd:element name="property" type="propertyType"/>
+        </xsd:choice>
+    </xsd:complexType>
+
+    <xsd:complexType name="connection-creation-optionsType">
+        <xsd:choice minOccurs="0" maxOccurs="unbounded">
+            <xsd:element name="property" type="propertyType"/>
+        </xsd:choice>
+    </xsd:complexType>
+
+    <xsd:complexType name="propertyType">
+        <xsd:attribute name="name" type="xsd:string" use="required">
+            <xsd:annotation>
+                <xsd:documentation>
+                    The name of the property. Example org.xnio.Options.SASL_POLICY_NOANONYMOUS
+                </xsd:documentation>
+            </xsd:annotation>
+        </xsd:attribute>
+        <xsd:attribute name="value" type="xsd:string" use="required">
+            <xsd:annotation>
+                <xsd:documentation>
+                    The value of the property.
+                </xsd:documentation>
+            </xsd:annotation>
+        </xsd:attribute>
+    </xsd:complexType>
+
+	<xsd:complexType name="profileType">
+		<xsd:attribute name="name" type="xsd:string" use="required">
+			<xsd:annotation>
+				<xsd:documentation>
+					The name of the profile.
+				</xsd:documentation>
+			</xsd:annotation>
+		</xsd:attribute>
+	</xsd:complexType>
+
```

Comment:
need to fix the indentation, eg. replace tab with space consistent with other elements.